src/components/Banners/MigrateEnsBanner/MigrateEnsBanner.js

```diff
@@ -0,0 +1,125 @@
+// @flow
+/*
+    Pillar Wallet: the personal data locker
+    Copyright (C) 2021 Stiftung Pillar Project
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+
+import React, { useState } from 'react';
+import styled from 'styled-components/native';
+import LinearGradient from 'react-native-linear-gradient';
+import { useTranslationWithPrefix } from 'translations/translate';
+import { useNavigation } from 'react-navigation-hooks';
+import { useSelector } from 'react-redux';
+
+// components
+import Text from 'components/modern/Text';
+import Icon from 'components/Icon';
+
+// constants
+import { ENS_MIGRATION_CONFIRM } from 'constants/navigationConstants';
+
+// selectors
+import { isEnsMigrationNeededSelector } from 'selectors/archanova';
+
+// utils
+import { appFont, fontStyles, spacing } from 'utils/variables';
+
+// types
+import type { ViewStyleProp } from 'utils/types/react-native';
+
+// assets
+const smartWalletIcon = require('assets/icons/smart-wallet-migrate.png');
+
+
+type Props = {|
+  style?: ViewStyleProp,
+|};
+
+const MigrateEnsBanner = ({ style }: Props) => {
+  const isEnsMigrationNeeded = useSelector(isEnsMigrationNeededSelector);
```

Comment:
minor:
`useRootSelector` from `'selectors'` would give you our store type checking